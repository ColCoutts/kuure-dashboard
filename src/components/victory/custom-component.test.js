import React from 'react';
import { shallow } from 'enzyme';
import CustomTheme from './custom-component';

describe('victory component', () => {
  it('renders a snapshot', () => {
    const wrapper = shallow(<CustomTheme />);
    expect(wrapper).toMatchSnapshot();
  });
});
