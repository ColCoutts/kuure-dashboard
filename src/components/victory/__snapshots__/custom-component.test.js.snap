// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`victory component renders a snapshot 1`] = `ShallowWrapper {}`;
